api/draft_nodes/permissions.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from osf.models import (
+    DraftRegistration,
+    DraftNode,
+)
+from api.draft_registrations.permissions import IsContributorOrAdminContributor
+
+from api.base.utils import assert_resource_type
+
+
+class ContributorOnDraftRegistration(IsContributorOrAdminContributor):
```

Comment:
Adapt to changes on api/draft_registration/permissions.py

Reply:
Look into possibly modifying api.nodes.permissions.ContributorOrPublic to not require a child permissions class.